### DISCUSSION TITLE
what the fuck happen to the config files!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by paperclip52** [lllyasviel/Fooocus#3125] [General]
what the fuck happen to the config files!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by lgwjames** [comment]
i would highly point out not to swear on a site

**Comment by mashb1t** [comment]
what happened to it? any constructive feedback or console output with an error?
